packages/framework/src/Controller/Admin/FilePickerController.php

```diff
@@ -24,83 +24,57 @@ class FilePickerController extends AdminBaseController
      * @param \Shopsys\FrameworkBundle\Model\Product\Listing\ProductListAdminFacade $productListAdminFacade
      * @param \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileFacade $uploadedFileFacade
      * @param \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFileAdminListFacade $uploadedFileAdminListFacade
+     * @param \Shopsys\FrameworkBundle\Component\UploadedFile\Grid\FilePickerGridFactory $uploadedFileGridFactory
+     * @param \Shopsys\FrameworkBundle\Component\UploadedFile\Grid\FilePickerGridFactory $filePickerGridFactory
      */
     public function __construct(
         protected readonly AdministratorGridFacade $administratorGridFacade,
         protected readonly GridFactory $gridFactory,
         protected readonly ProductListAdminFacade $productListAdminFacade,
         protected readonly UploadedFileFacade $uploadedFileFacade,
         protected readonly UploadedFileAdminListFacade $uploadedFileAdminListFacade,
```

Comment:
as well as these two dependencies are not necessary here anymore

Reply:
fixed